packages/client/src/modules/token-set.ts

```diff
@@ -1,6 +1,6 @@
 import { TokenResponse } from '../api';
 import { nowRoundToSec } from '../utils';
-import { decodeToken, IDToken } from '../utils/id-token';
+import { decodeIdToken, IdTokenClaims } from '../utils/id-token';
```

Comment:
should add `@` alias?

Reply:
Ok, I will do it later in another PR. [issue](https://linear.app/silverhand/issue/LOG-1275/use-alias-path-in-js-project)

Reply:
you can directly use LOG-1275 to refer since we have auto-link configured

Reply:
Nice feature. Got it.